src/helpers.js

```diff
@@ -45,15 +78,18 @@ function getAbellConfigs() {
     delete require.cache[path.join(process.cwd(), 'abell.config.js')];
     abellConfig = require(path.join(process.cwd(), 'abell.config.js'));
     if (Object.keys(abellConfig).length <= 0) {
-      throw new Error('Something went wrong while fetching new configurations. Save again to refresh the dev server.'); // eslint-disable-line
+      throw new Error(
+        `Something went wrong while fetching new configurations. 
+         Save again to refresh the dev server.`
+      ); // eslint-disable-line
```

Comment:
This will add an extra line break in terminal too I guess, since it is using ` for strings. Writing it in single line is fine since it is just an error message and not an important code.

Reply:
This won't add a new line unless a \n is added there. I split that line because eslint was complaining of the line being too long. Do you have any suggestions for that ?

Reply:
![image](https://user-images.githubusercontent.com/30949385/82157296-32a2b480-989e-11ea-9e6a-0b6f3ce28036.png)
It actually does. In JavaScript when you use '\`' for strings, the keep the line breaks and additional spaces. This is one of the reason why I had // eslint-disable-line their to disable the eslint error. And in case of strings it's fine if it crosses the line length limit because the workaround of adding `\n` to satisfy eslint is more unreadable than having long horizontal line

Reply:
Fixed this ✅